docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -272,6 +272,14 @@ use openzeppelin::presets::Account;
 
 Basic account contract leveraging xref:#AccountComponent[AccountComponent].
 
+:class-hashes: xref:presets.adoc[Compiled class hash]
+
+[.contract-index]
+.{class-hashes}
+--
+0x016c6081eb34ad1e0c5513234ed0c025b3c7f305902d291bad534cd6474c85bc
```

Comment:
is there any way of abstracting these into variables that are reused across the docs? maybe [early optimization](https://softwareengineering.stackexchange.com/questions/80084/is-premature-optimization-really-the-root-of-all-evil) but I think if we update these often we could make sure we're updating it properly everywhere